jekyll/_cci2/skip-build.md

```diff
@@ -1,27 +1,35 @@
 ---
 layout: classic-docs
-title: Skip and Cancel Builds
-short-title: Skip and Cancel Builds
-description: How to Prevent CircleCI From Automatically Building
+title: Skip or Cancel Pipelines
+description: This document describes how to prevent automatically building your project.
 order: 100
 version:
 - Cloud
 - Server v3.x
 - Server v2.x
 ---
 
-This document describes how to skip or cancel builds in the following sections.
+This document describes how to skip or cancel pipelines. There are a couple of ways to do this. A pipeline can be skipped on commit, or by using the auto-cancel feature. Both methods are described below.
 
 * TOC
 {:toc}
 
-## Skipping a build
+## Skipping a pipeline
 {: #skipping-a-build }
 
-By default, CircleCI automatically builds a project whenever you push changes to a version control system (VCS). You can override this behavior by adding a `[ci skip]` or `[skip ci]` tag within the first 250 characters of the body of the commit or the commit's title. This not only skips the marked commit, but also **all other commits** in the push.
+By default, CircleCI automatically runs a pipeline whenever you push changes to your project. You can override this behavior by adding a `[ci skip]` or `[skip ci]` tag within the first 250 characters of the body of the commit or the commit's title. This not only skips the marked commit, but also **all other commits** in the push.
 
-**Note:**
-This feature is not supported for fork PRs. Scheduled workflows will not be cancelled even if you push a commit with `[ci skip]` message. Changing the config file is the way to upgrade the current schedule.
+**CircleCI server v2.x**
+If you are using CircleCI server v2.x, you can still use the method for skipping workflows described here, even though you are not using the pipelines feature.
+
+### Scope
+{: #scope }
+
+A few points to note regarding the scope of the `ci skip` feature:
+
+* The pipeline and workflows will still exist for these commits but all jobs will **not** be run.
```

Comment:
That's a bit awkward. How about "no jobs will be run"?

Reply:
😆 yes thanks for pointing this out!